pyproject.toml

```diff
@@ -14,11 +14,12 @@ babel = "==2.*,>=2.0.0,>=2.5.1"
 backoff = "==1.*,>=1.0.0,>=1.4.3"
 "backports.tempfile" = ">=1.0rc1"
 beard = "==0.*,>=0.0.0,>=0.2.0"
-celery = "<4.2.0,==4.*,>=4.0.0,>=4.1.0"
+celery = "<4.5.0,==4.*,>=4.4.0"
 click = "<7.0,==6.*,>=6.7.0"
 click-spinner = "==0.*,>=0.0.0,>=0.1.8"
 coverage = {version = "==4.*,>=4.0.0,>=4.5.4", optional = true}
 docutils = {version = "<0.16,==0.*,>=0.15.0,>=0.15.2", optional = true}
+elasticsearch = "<=7.2.0"
```

Comment:
isn't it dangerous? shall we add ```>=7.1.0,<7.2.0```

Reply:
elasticsearch major version is managed by invenio-search.
`invenio-search = {version = "~=1.2.3,<1.3.0", extras = [ "elasticsearch7",]}`